installer/requirements.txt

```diff
@@ -11,3 +10,4 @@ toml==0.10.0
 wheel
 pyinstaller
 setuptools<45.0.0  # https://github.com/pypa/setuptools/issues/1963
+https://pyyaml.org/download/pyyaml/PyYAML-5.3.1-cp38-cp38-win_amd64.whl
```

Comment:
Could we add the `sys_platform == 'win32'` flag so pyinstaller can produce a binary for linux as well?

Reply:
I'll try it again, but it was doing something weird with GH actions.  You can always do `python3 ./setup.py install` to achieve that (it's what I've been doing).